src/datasets/commands/dummy_data.py

```diff
@@ -83,6 +83,7 @@ def auto_generate_dummy_data_folder(
                 self.mock_download_manager.dummy_data_folder,
```

Comment:
can you remove the changes in this file please ?

Looks like `black` wanted to do some changes while it's actually fine (line length < 119)

Reply:
Done